token-metadata/program/src/utils.rs

```diff
@@ -963,6 +973,19 @@ pub fn process_create_metadata_accounts_logic(
 
     let mut metadata: Metadata = Metadata::from_account_info(metadata_account_info)?;
     let compatible_data = data.to_v1();
+
+    // This allows the Bubblegum program to create metadata with verified creators since they were
+    // verified already by the Bubblegum program.
+    const BUBBLEGUM_ACTIVATED: bool = false;
```

Comment:
This is here because we cannot have this existing yet in any real deployed code, because Bubblegum is not updated to fully check signers for setting the verify flag.  So it is being added as a disabled feature.  Does that answer your question?